CWE/inventions/New inventions.txt

```diff
@@ -300,4 +300,465 @@ full_body_replacement = {
+		modifier = {
+			factor = 1
+			beijing_consensus = 1
+		}
+	}
+	effect = {
+		shared_prestige = 15
+		factory_goods_output = {
+			culture = 0.01
+		}
+	}
+}
+
+
+new_left = {
+	limit = { wars_of_national_liberation = 1 } #Ideological Mass Mobilization
+	chance = {
+		base = 1
+		modifier = {
+			factor = 1
+			OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 government = socialist_democracy }
+		}
+		modifier = {
+			factor = 1
+			OR = { continent = europe continent = north_america }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}
+		modifier = {
+			factor = -2
+			NOT = { year = 1960 }
+		}
+		modifier = {
+			factor = 1
+			has_country_flag = eurocommunism
+		}
+	}
+	effect = {
+		social_reform_desire = 0.01
+		rebel_org_gain = {
+			faction = communist_rebels
+			value = 0.05
+		}
+	}
+}
+
+eurocommunism_inv = { 
+	limit = { seventies_subcultures  = 1 } #Progressivism
+	chance = {
+		base = 1
+		modifier = {
+			factor = 1
+			OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 government = socialist_democracy }
+		}
+		modifier = {
+			factor = -1
+			NOT = { continent = europe }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}
+		modifier = {
+			factor = 3
+			has_country_flag = eurocommunism
+		}
+	}
+	effect = {
+		social_reform_desire = 0.01
+	}
+}
+
+human_immunocontraception = { 
+	limit = { genomics = 1 } #AI Guided Manufacturing
+	chance = {
+		base = 1
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+	}
+	effect = {
+        	pop_growth = -0.0001
+	}
+}
+
+reversible_inhibition = {
+	limit = { clinical_pharmacology = 1 } #Advanced Biotechnology
+	chance = {
+		base = 1
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+	}
+	effect = {
+        	pop_growth = -0.0001
+	}
+}
+
+sustainable_population = {
+	limit = { globalization = 1 } #Political Globalisation
+	chance = {
+		base = 0
+		modifier = {
+			factor = -1
+			NOT = { year = 1995 }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+		modifier = {
+			factor = 2
+			OR = { 
+				has_country_modifier = overpopulated_nation1
+				has_country_modifier = overpopulated_nation2
+				has_country_modifier = overpopulated_nation3
+				has_country_modifier = overpopulated_nation4
+				has_country_modifier = overpopulated_nation5
+				has_country_modifier = overpopulated_nation6
+				has_country_modifier = overpopulated_nation7
+				has_country_modifier = overpopulated_nation8
```

Comment:
overpopulated_nation is a triggered modifier and not a country modifier

Reply:
Does it count them differently, for some reason? How annoying. Fortunately, its only in there, and that invention is getting axed. But its good to know for the future.